grpc-proto/proto/query.proto

```diff
@@ -162,13 +162,15 @@ message QueryParameters {
   google.protobuf.StringValue keyspace = 1;
 
   // The consistency level.
+  // If unset, it defaults to ONE.
   ConsistencyValue consistency = 2;
 
   // The maximum number of rows that will be returned in the response. If there are more results,
   // the response will contain a paging state, and additional queries will be needed to retrieve
   // subsequent pages.
   // Note that this is only a suggestion, the server reserves the right to return slightly smaller
   // or bigger pages.
+  // If unset, the request won't page.
```

Comment:
From the `Parameters` javadoc.
That sounds dangerous, I think we should enforce one in the service.

Reply:
#1156